src/hotspot/share/utilities/bitMap.inline.hpp

```diff
@@ -228,6 +228,66 @@ inline BitMap::idx_t BitMap::get_next_bit_impl(idx_t l_index, idx_t r_index) con
   return r_index;
 }
 
+template<BitMap::bm_word_t flip, bool aligned_left>
+inline BitMap::idx_t BitMap::get_prev_bit_impl(idx_t l_index, idx_t r_index) const {
+  STATIC_ASSERT(flip == find_ones_flip || flip == find_zeros_flip);
+  verify_range(l_index, r_index);
+  assert(!aligned_left || is_aligned(l_index, BitsPerWord), "l_index not aligned");
+  assert(sizeof(intptr_t) == sizeof(bm_word_t), "must be for the high bit check");
+
+  // The first word often contains an interesting bit, either due to
+  // density or because of features of the calling algorithm.  So it's
+  // important to examine that first word with a minimum of fuss,
+  // minimizing setup time for later words that will be wasted if the
+  // first word is indeed interesting.
+
+  // The benefit from aligned_left being true is very small. It saves
+  // an operation when returning results.
+
+  // The return value of l_index when no bit is found is BitMap::NotFound.
+
+  if (l_index <= r_index) {
```

Comment:
This condition is always true if the `verify_range` precondition is met.  This is an artifact of making `r_index` inclusive rather than exclusive.